hyp3_metadata/readme.md.txt.j2

```diff
@@ -7,14 +7,15 @@ Processing Date/Time: {{ processing_date.isoformat(timespec='seconds') }}
 
 The folder and each of its contents all share the same base name, using the following convention:
 ```
-S1x_yy_aaaaaaaaTbbbbbb_ppo_RTCzz_G_defklm_ssss
+S1x_yy_aaaaaaaaTbbbbbb_ppo_RTCzz_u_defklm_ssss
 x:          Sentinel-1 Mission (A or B)
 yy:         Beam Mode
 aaaaaaaa:   Start Date of Acquisition (YYYYMMDD)
 bbbbbb:     Start Time of Acquisition (HHMMSS)
 pp:         Polarization
 o:          Orbit Type: Precise (P), Restituted (R), or Original Predicted (O)
 zz:         Terrain Correction Resolution
+u:          Software Package Used: GAMMA (G)
```

Comment:
Nothing critical, but this reads a little weird to me, since we're using `u` as "Unmasked" in a couple lines -- I thought we were trying to not repeat letters. 

I think i'd just do:

```
G: GAMMA Software Package Used
```

Reply:
While there was an effort not to reuse the option letters, and there aren't letters re-used in the filename schema, I don't think there was as much effort in not reusing letters across the two. The letters used in the filename schema were generally alphabetical by element to start with, but as the schema was refined, some sections were moved, and others removed entirely; the u remains from the first schema proposal as the placeholder for processor type. 

In looking it over, it doesn't look like we use q anywhere; we could switch to that. But if we change it, we should change it in the InSAR documentation, too. 

Because we'll be using different software eventually, and users may have products of the same type generated by multiple different processors, I think it keeps it more generic to keep the schema the same. If I'm in the minority on this, however, we could change it to be processor-specific.

![image](https://user-images.githubusercontent.com/61886203/97929324-615dfb80-1d15-11eb-8fd7-5b51ace950b4.png)

Reply:
It's mostly the proximity of the `u`s to each other here, and that it's not a multi-valued field for this product so we don't really need the placeholder. 

Doing it the way I wrote would punt any discussion on the file-name schema until we want to publish that. 

But it's a big problem -- just something that jumped out as a possible source of confusion.

Reply:
*not; typo